### PR TITLE
Document testing code from rawhide branch

### PR DESCRIPTION
**Pull request by jan-cerny**
Testing code on rawhide branch is special and common methods like reanaconda don't work. We will document steps to facilitate the testing and review of the rawhide branch in the Developer guide. We have discovered this during our review of https://github.com/OpenSCAP/oscap-anaconda-addon/pull/242

### CODE REVIEW

File: docs/manual/developer_guide.adoc
Comment (matejak) [inline docs/manual/developer_guide.adoc:223]:
```suggestion
+
where the IP address may vary - if the update image is not downloaded during the installation media boot (you can tell that something happened from the output of the Python server command), look up the gateway IP address from the respective VM configuration.
+
```

**Comment by matejak** [comment]
LGTM, feel free to self-merge if you like my addition.
BTW is this the latest master? I would expect that #238 fixed the imports, and hence the unit tests.

**Comment by jan-cerny** [comment]
> LGTM, feel free to self-merge if you like my addition.

@matejak I can't self-merge, GitHub requires a reviewer's approval.

> BTW is this the latest master? I would expect that https://github.com/OpenSCAP/oscap-anaconda-addon/pull/238 fixed the imports, and hence the unit tests.

@matejak For me it seems that it did the opposite - the PR #238  changed the imports to `pyanaconda.core.threads` and the unit tests traceback shows that `pyanaconda.core.threads` doesn't exist.

**Comment by jan-cerny** [comment]
@matejak  PING

**Comment by jan-cerny** [comment]
@matejak PING

**Comment by jan-cerny** [comment]
@matejak see above

**Review by matejak — APPROVED** [review]
LGTM

**Comment by matejak** [comment]
I can't rerun the tests for some reasons, I would expect that they pass on this more recent rawhide.

**Pull request merged**
